docs/index.rst

```diff
@@ -257,6 +257,35 @@ disabled individually.
 See the :class:`~urllib3.util.retry.Retry` definition for more details.
 
 
+Stream
+------
+
+You may also stream your response and get data as they come (e.g. when using
+``transfer-encoding: chunked``). In this case, method
+:func:`~urllib3.response.HTTPResponse.stream` will return generator.
+
+::
+
+    >>> import urllib3, datetime
```

Comment:
In the future, lean towards simpler examples (e.g. no need for datetime).

Also maybe use http://httpbin.org/ rather than a gist/localhost reference. :)